services/movie.py

```diff
@@ -0,0 +1,39 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(genres_ids: list = None, actors_ids: list = None) -> QuerySet:
+    if genres_ids is None and actors_ids is None:
+        return Movie.objects.all()
+
+    if genres_ids is not None and actors_ids is not None:
+        return Movie.objects.filter(
+            actors__id__in=actors_ids, genres__id__in=genres_ids
+        )
+
+    if genres_ids is not None and actors_ids is None:
+        return Movie.objects.filter(genres__id__in=genres_ids)
+
+    if genres_ids is None and actors_ids is not None:
```

Comment:
I think we can do that with a little less number of checks

Reply:
Used query